lesson05/assignment/css/style.scss

```diff
@@ -4,3 +4,164 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// Common Font Sizes
+$font-verdana: Verdana, Geneva, Tahoma, sans-serif;
+$font-tahoma: Tahoma, sans-serif;
+$font-small: .6rem;
+$font-medium: 1rem;
+$font-large: 1.5rem;
+$font-veryLarge: 2.0rem;
+
+.wrapper {
+  display: grid;
+  width: 100%;
+  grid-template-columns: 25% 25% 25% 25%;
+  grid-template-rows: 0fr .2fr .2fr; 
+}
```

Comment:
You will want to use media queries and adjust your grid for different screen sizes